src/gcp_scanner/crawl.py

```diff
@@ -403,10 +386,7 @@ def get_managed_zones(project_name: str,
     request = service.managedZones().list(project=project_name)
     while request is not None:
       response = request.execute()
-
-      for managed_zone in response["managedZones"]:
-        zones_list.append(managed_zone)
-
+      zones_list = response["managedZones"]
```

Comment:
Use `.get(...)` to avoid exceptions just like in the other places

Reply:
used response.get("managedZones",[]) instead of directly referencing the key.